cassandra/delete_data.py

```diff
@@ -11,12 +11,17 @@
 import os
 import sys
 
-# Add parent directory to python path to locate the metadata_service_api package
-sys.path.insert(0, '/'.join((os.path.dirname(os.path.realpath(__file__)), '..')))
+# Add parent directory to python path to locate the
+# metadata_service_api package
+sys.path.insert(0, '/'.join(
+        (os.path.dirname(os.path.realpath(__file__)), '..')))
```

Comment:
improper indentation - PEP-8 - consider using PyCharm or another editor that conforms to PEP8 coding standards
Also - please do not make unnecessary format changes (this repeats throughout the file)